src/docker.js

```diff
@@ -132,7 +212,7 @@ docker.buildImage = function(project, tarPath, imageId, buildId, buildLogger, do
  */
 docker.tag = function(imageId, buildId, branch) {
   var deferred  = Q.defer();
-  var image     = docker.client.getImage(buildId);
+  var image     = docker.client.getImage(imageId + ':' + branch);
```

Comment:
This fix a race condition, when roger builds `latest` and a branch / tag at the same time, the images pushed are all `latest`

Reply:
:+1: